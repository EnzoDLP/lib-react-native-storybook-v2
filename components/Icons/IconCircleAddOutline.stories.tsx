import type { Meta, StoryObj } from '@storybook/react';
import React from 'react';
import { View } from 'react-native';
import { IconCircleAddOutline } from './IconCircleAddOutline.component';

const meta = {
  title: 'Icons/IconCircleAddOutline',
  component: IconCircleAddOutline,
  decorators: [
    (Story) => (
      <View style={{ padding: 16 }}>
        <Story />
      </View>
    ),
  ],
} satisfies Meta<typeof IconCircleAddOutline>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {
    accessibilityLabel: 'Ic√¥ne ajouter',
    color: '#000000',
    size: 24,
  },
};
