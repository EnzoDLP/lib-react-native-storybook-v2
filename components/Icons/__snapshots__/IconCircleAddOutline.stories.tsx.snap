// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons/IconCircleAddOutline Default smoke-test 1`] = `
<div class="css-view-175oi2r"
     style="padding: 16px;"
>
  <svg width="24"
       height="24"
       viewbox="0 0 24 24"
       fill="none"
       aria-label="IcÃ´ne ajouter"
       role="img"
  >
    <g clip-path="url(#clip0_19238_203083)">
      <path d="M12 0a12 12 0 1012 12A12.013 12.013 0 0012 0zm0 22a10 10 0 1110-10 10.011 10.011 0 01-10 10zm5-10a1 1 0 01-1 1h-3v3a1 1 0 01-2 0v-3H8a1 1 0 010-2h3V8a1 1 0 012 0v3h3a1 1 0 011 1z"
            fill="#000000"
      >
      </path>
    </g>
  </svg>
</div>
`;
